packages/web/src/common/store/pages/signon/reducer.js

```diff
@@ -85,7 +85,7 @@ const initialState = {
     categories: {},
     selectedUserIds: []
   },
-  genres: ['Hip-Hop/Rap'],
```

Comment:
yikes good find

Reply:
yeah I was wondering why I kept getting routed to the artists page immediately 😅